src/BenchmarksServer/Startup.cs

```diff
@@ -217,6 +221,28 @@ public static int Main(string[] args)
                                 Debug.Assert(process == null);
                                 process = StartProcess(hostname, database, sqlConnectionString, Path.Combine(tempDir, benchmarksDir),
                                     job, dotnetHome);
+                                
+                                if (timer != null)
+                                {
+                                    timer.Dispose();
+                                }
+
+                                DateTime lastMonitorTime = DateTime.UtcNow;
+                                var oldCPUTime = new TimeSpan(0);
+
+                                timer = new Timer(_ => 
+                                {
+
+                                    var newCPUTime = process.TotalProcessorTime;
+                                    var ellapsed = DateTime.UtcNow.Subtract(lastMonitorTime).TotalMilliseconds;
+                                    var cpu =  Math.Round((newCPUTime - oldCPUTime).TotalMilliseconds / (Environment.ProcessorCount *  ellapsed) * 100);
+                                    lastMonitorTime = DateTime.UtcNow;
```

Comment:
Technically I think you should only call DateTime.UtcNow once and assign it to a local var, like you do with `newCPUTime`.  Otherwise you could miss some elapsed time between the two calls to DateTime.UtcNow.